lib/zendesk_apps_support/validations/manifest.rb

```diff
@@ -416,6 +429,12 @@ def no_template_format_error(manifest)
             ValidationError.new(:invalid_no_template)
           end
         end
+
+        def validate_url(value, label_for_error)
+          unless value.nil? || value.match(/^https?:\/\//)
```

Comment:
Have you confirmed with product that we require https urls now? There's a lot of stuff like `http://foo.com`, `www.foo.com` and just `foo.com`s in the db at the moment.

Reply:
interesting, i check that if the url is www.xxxx.com, the redirection will treat that as a path of the url
in my case
```
"url": "www.urlencoder.org"
```
redirect me to: `https://z3nchristinawu.zendesk-staging.com/support/apps/www.urlencoder.org`

<img width="780" alt="image" src="https://user-images.githubusercontent.com/42607545/168738768-151bf422-142d-489e-aef3-dafcc3e89a36.png">

that mean we dont support that syntax very well?

Reply:
currently `/^https?:\/\//` match both `http://` and `https:`

I wonder if we need to restrict them to `https` only

Reply:
Thanks @mmassaki , yep, we found many http and https in the database, so we should keep both

Reply:
> Use URI class to validate urls

Done